Dockerfile

```
@@ -1,14 +1,31 @@
-FROM node:22-alpine
+# Stage 0: Build application
+FROM node:22-alpine AS build
 WORKDIR /app
 
 # Install dependencies from package-lock.json
-COPY ["package*.json", "."]
+COPY package*.json ./
 RUN npm ci
 
 # Build from source
 COPY . .
 RUN npm run build
 
+
+# Stage 1: Create production image
+FROM node:22-alpine AS production
+WORKDIR /app
+
+# Install only production dependencies
+COPY package*.json ./
+ENV NODE_ENV production
+RUN npm ci --omit=dev
```

```suggestion
RUN npm ci --omit dev
```
Wenn ich `npm ci --help` mache, steht da nicht, dass da ein `=` hinkommt. Kann trotzdem funktionieren, aber hätte ich lieber nicht.

---

Ja true, geht beides, aber so ist schöner.